src/sentry/mediators/sentry_app_installations/creator.py

```diff
@@ -44,6 +45,16 @@ def _create_api_grant(self):
             application=self.api_application,
         )
 
+    def _create_service_hooks(self):
+        for project in self.organization.project_set.all():
+            service_hooks.Creator.run(
+                application=self.api_application,
+                actor=self.install,
+                project=project,
+                events=self.sentry_app.events,
```

Comment:
Where do we validate the event names are sane?

Reply:
Added https://github.com/getsentry/sentry/pull/10807/files#diff-e459b195eadd3751bd5fc6ffaa92e3abR21